Source/DafnyRuntime/DafnyRuntimeRust/src/lib.rs

```diff
@@ -30,275 +30,1333 @@ pub type SizeT = usize;
 // We use the named version using {...}, and use snake_case format
 
 // The T must be either a *const T (allocated) OR a Reference Counting (immutable)
+pub mod dafny_runtime_conversions {
+    use crate::DafnyType;
+    use crate::DafnyTypeEq;
+    pub type DafnyInt = crate::DafnyInt;
+    pub type DafnySequence<T> = crate::Sequence<T>;
+    pub type DafnyMap<K, V> = crate::Map<K, V>;
+    pub type DafnySet<T> = crate::Set<T>;
+    pub type DafnyMultiset<T> = crate::Multiset<T>;
+    pub type DafnyBool = bool;
+    pub type DafnyString = String;
```

Comment:
Good point. I removed this alias because it was used nowhere and it's a wrong one. The aliases for strings are in the respective submodules `unicode_chars_true::DafnyString` and `unicode_chars_false::DafnyString`